src/jsx.d.ts

```diff
@@ -753,6 +754,10 @@ export namespace JSXInternal {
 		wmode?: string;
 		wrap?: string;
 
+		// Non-standard Attributes
+		autocapitalize?: string;
+		autoCapitalize?: string;
```

Comment:
```suggestion
		autocapitalize?: 'off' | 'none' | 'on' | 'sentences' | 'words' | 'characters';
		autoCapitalize?: 'off' | 'none' | 'on' | 'sentences' | 'words' | 'characters';
```

Types could be improved by being more specific. As the value has to be one of those values, `string` is very vague by comparison. [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize)

Reply:
very good point